[2024-04-09T10:10:42.557+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-09T10:10:42.683+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_1.create_about_us_about_txt_file manual__2024-04-09T10:10:15.020936+00:00 [queued]>
[2024-04-09T10:10:42.762+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_1.create_about_us_about_txt_file manual__2024-04-09T10:10:15.020936+00:00 [queued]>
[2024-04-09T10:10:42.766+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-04-09T10:10:42.955+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): create_about_us_about_txt_file> on 2024-04-09 10:10:15.020936+00:00
[2024-04-09T10:10:43.007+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1765) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-09T10:10:43.013+0000] {standard_task_runner.py:63} INFO - Started process 1777 to run task
[2024-04-09T10:10:43.018+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_1', 'create_about_us_about_txt_file', 'manual__2024-04-09T10:10:15.020936+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/dag-1.py', '--cfg-path', '/tmp/tmp_on6wk1t']
[2024-04-09T10:10:43.037+0000] {standard_task_runner.py:91} INFO - Job 29: Subtask create_about_us_about_txt_file
[2024-04-09T10:10:45.462+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_1.create_about_us_about_txt_file manual__2024-04-09T10:10:15.020936+00:00 [running]> on host b6bc23c6abe9
[2024-04-09T10:10:46.348+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_1' AIRFLOW_CTX_TASK_ID='create_about_us_about_txt_file' AIRFLOW_CTX_EXECUTION_DATE='2024-04-09T10:10:15.020936+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-09T10:10:15.020936+00:00'
[2024-04-09T10:10:46.393+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-09T10:10:46.395+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-09T10:10:46.397+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'mkdir -p about_us && touch about_us/about.txt']
[2024-04-09T10:10:46.466+0000] {subprocess.py:86} INFO - Output:
[2024-04-09T10:10:46.512+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-09T10:10:46.513+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-09T10:10:46.574+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_1, task_id=create_about_us_about_txt_file, execution_date=20240409T101015, start_date=20240409T101042, end_date=20240409T101046
[2024-04-09T10:10:46.701+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-09T10:10:46.757+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-09T10:10:46.773+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
